css/helpers/_mixin.scss

```diff
@@ -0,0 +1,30 @@
+@mixin square($width, $height: $width) {
+    width: $width;
+    height: $height;
+}
+
+@mixin border($size, $type, $color) {
+    border: $size $type $color;
+}
+
+$mobile-screen: 480px;
+$tablet-screen: 768px;
+$desktop-screen: 1280px;
+ 
+@mixin mobile-screen() {
```

Comment:
plusik za podjecie proby, dziala poprawnie ale na zajeciach pokazemy sobie jeszcze lepszy sposob jak to ograc